### DISCUSSION TITLE
How to download HACS into BBN?

### DISCUSSION THREAD

**Discussion by PtJudeRI** [bareboat-necessities/lysmarine_gen#404] [Q&A] [answered]
Hi All,
I am using the HomeAssistant software to add in Zigbee sensors to my boat.  Temp/Humidity/Bilge/Battery.  The basic integration there has worked fine, but I also want to add in some GPS enabled features.  It seems that I need to have additional modules installed from the HACS repository, but when I go to install HACS (the community store) using this code:

wget -O - https://get.hacs.xyz | bash -

I receive an error:

“ Trying to find the correct directory
Could not find the directory for Home Assistant.
Manually change the directory to the root of your Home Assistant configuration
With the user that is running Home Assistant
and run the script again”

Is there a workaround here or am I SOL based off of the structure of the OS?

**Comment by mgrouch** [comment, marked as answer]
You need to be a homeassistant user in shell
for running that command

````
sudo su
su - homeassistant

````

**Reply by PtJudeRI** [reply]
Thank you!  I will give this a shot when I get to my system.  Ideally, I would like to be able to have either HA or BBN automate a boat position announcement that will deliver an SMS at regular intervals when I am offshore.  I have a Starlink tied into the system, so the connection is there; just need position and the software to jump in.

**Reply by mgrouch** [reply]
For power consumption reasons for that feature you are better of with esp32 based solution. 

I’ve started with one simple example here. 
if you comfortable with coding esp32 it’s not hard to add gps sensor to it. 

https://github.com/bareboat-necessities/bbn-m5atomS3-lite/tree/main/bbn_m5atomS3_lite_whatsapp
